.gitignore

```diff
@@ -30,3 +30,5 @@ typings
 yarn.lock
 package-lock.json
 build
+dist/**/*.d.ts
```

Comment:
Do we want to ignore the type files?

Reply:
Don't Know, most of the library does it. They are mainly useful when you develop with a package manager